runtime/j9vm/jvm.c

```diff
@@ -2170,26 +2213,35 @@ JNI_CreateJavaVM_impl(JavaVM **pvm, void **penv, void *vm_args, BOOLEAN isJITSer
 		case J9PORT_ERROR_INIT_WRONG_MAJOR_VERSION: {
 			portGetVersionFn(&j9portLibrary, &actualVersion);
 
-			fprintf(stderr,"Error: Port Library failed to initialize: expected major version %u, actual version is %u\n",
-					portLibraryVersion.majorVersionNumber, actualVersion.majorVersionNumber);
+			fprintf(
+				stderr,
+				"Error: Port Library failed to initialize: expected major version %u, actual version is %u\n",
+				portLibraryVersion.majorVersionNumber,
+				actualVersion.majorVersionNumber);
```

Comment:
My preference is that continued lines are indented two extra levels (like line 2174 was).